bot.py

```diff
@@ -61,16 +63,20 @@ async def answer(message, embed):
     except discord.errors.Forbidden:
         log.warning(
             f'[{message.guild.name}][{message.channel}] Could not post response, channel is forbidden for me.')
+    except discord.errors.HTTPException:
+        log.warning(
+            f'[{message.guild.name}][{message.channel}] Could not post response, message exceeds maximum size of 6000.')
```

Comment:
oh that's broad. Catching all HTTP Exceptions, but giving size limit error, without exception text?

Reply:
I removed the handler. Let's add a handler for this later